appveyor.yml

```diff
@@ -11,5 +11,41 @@ skip_tags: true
 build:
   project: ReferenceAssemblyAnnotator.sln
   verbosity: minimal
+test_script:
+- git clean -dxf tests
+- msbuild /restore tests/MultiTFM /warnaserror /v:m
+- git clean -dxf tests
+- msbuild /restore tests/SingleTFM /p:TestFramework=net35 /warnaserror /v:m
+- git clean -dxf tests
+- msbuild /restore tests/SingleTFM /p:TestFramework=net48 /warnaserror /v:m
+- git clean -dxf tests
+- msbuild /restore tests/SingleTFM /p:TestFramework=netstandard1.6 /warnaserror /v:m
+- git clean -dxf tests
+- msbuild /restore tests/SingleTFM /p:TestFramework=netstandard2.0 /warnaserror /v:m
+- git clean -dxf tests
+- msbuild /restore tests/SingleTFM /p:TestFramework=netcoreapp1.0 /p:DisableImplicitNuGetFallbackFolder=true /warnaserror /v:m
+- git clean -dxf tests
+- msbuild /restore tests/SingleTFM /p:TestFramework=netcoreapp1.1 /p:DisableImplicitNuGetFallbackFolder=true /warnaserror /v:m
+
+- git clean -dxf tests
+- dotnet msbuild -restore tests/MultiTFM -warnaserror -v:m
+- taskkill /im dotnet.exe /f
+- git clean -dxf tests
+- dotnet msbuild -restore tests/SingleTFM -p:TestFramework=net35 -warnaserror -v:m
+- taskkill /im dotnet.exe /f
+- git clean -dxf tests
+- dotnet msbuild -restore tests/SingleTFM -p:TestFramework=net48 -warnaserror -v:m
+- taskkill /im dotnet.exe /f
+- git clean -dxf tests
+- dotnet msbuild -restore tests/SingleTFM -p:TestFramework=netstandard1.6 -warnaserror -v:m
+- taskkill /im dotnet.exe /f
+- git clean -dxf tests
+- dotnet msbuild -restore tests/SingleTFM -p:TestFramework=netstandard2.0 -warnaserror -v:m
+- taskkill /im dotnet.exe /f
+- git clean -dxf tests
+- dotnet msbuild -restore tests/SingleTFM -p:TestFramework=netcoreapp1.0 -p:DisableImplicitNuGetFallbackFolder=true -warnaserror -v:m
+- taskkill /im dotnet.exe /f
+- git clean -dxf tests
+- dotnet msbuild -restore tests/SingleTFM -p:TestFramework=netcoreapp1.1 -p:DisableImplicitNuGetFallbackFolder=true -warnaserror -v:m
```

Comment:
@sharwell 

I was looking for something I could queue up from the command line. Do you mind if I add back a ps1 or cmd? `git clean -fxd test` was something I was thinking about.

`/nodeReuse:false` or env `MSBUILDDISABLENODEREUSE=1` can be used rather than taskkill IIRC.

Reply:
They are also recommended for CI:

> But since it's important to tell people with CI servers and so on to disable node reuse, we should have clear documentation in the usual place.

https://github.com/Microsoft/msbuild/issues/2916

Reply:
They are also recommended for CI:

> But since it's important to tell people with CI servers and so on to disable node reuse, we should have clear documentation in the usual place.

https://github.com/Microsoft/msbuild/issues/2916

Reply:
> But since it's important to tell people with CI servers and so on to disable node reuse, we should have clear documentation in the usual place.

This only applies to build servers that share state across builds. No CI server should ever be doing this (and neither AppVeyor nor Travis have this problem).